test/unit/spec/nbextensions/bulkImportCell/tabs/configure-spec.js

```diff
@@ -8,11 +8,16 @@ define([
 ], (ConfigureTab, Jupyter, Runtime, Props, Spec, TestAppObject) => {
     'use strict';
 
-    xdescribe('test the bulk import cell configure tab', () => {
+    describe('test the bulk import cell configure tab', () => {
         beforeAll(() => {
             Jupyter.narrative = {
                 getAuthToken: () => 'fakeToken',
             };
+
+            const params = TestAppObject.params;
+            TestAppObject.params = {
+                fastq_reads: params,
+            };
```

Comment:
the params were set up wrong, so the configure tab was generating a load of errors